include/boost/geometry/strategies/spherical/disjoint_segment_box.hpp

```diff
@@ -72,6 +72,73 @@ struct segment_box_spherical
 };
 
 
+struct segment_box_spherical_with_info
```

Comment:
For consistency: `spherical_segment_box_with_info`.

Reply:
ok